api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -46,14 +46,26 @@ POST /websiteUploader/upload
 
     Should Be Equal As Strings    ${response.json()['address']}    ${WEBSITE_ADDRESS}
 
-GET /all/domains 
-    ${response}=    GET
-    ...    ${API_URL}/all/domains
-    ...    expected_status=${STATUS_OK}
-    Should Contain    ${response.text}    flappy
-
 GET /my/domains/{nickname}
     ${response}=    GET
     ...    ${API_URL}/my/domains/${WALLET_NICKNAME}
     ...    expected_status=${STATUS_OK}
     Should Contain    ${response.text}    ${website_url}
+    
+POST /cmd/executeFunction To remove the dns entry
+    ${argument}=    keywords.String To Arg    ${website_url}
+    ${data}=    Create Dictionary
+    ...    nickname=${WALLET_NICKNAME}
+    ...    name=deleteEntryFromDNS
+    ...    at=AS12tsy5TtDyGsWggDvqDLagxZqkuTiTPqWhqmrx7KSQNJwQSbnBA
```

Comment:
this variable should not be hardcoded here: define a variable at the beginning of the file

idealy, we should call endpoint /network and save as a global variable the DNS address